README.md

```diff
@@ -1,2 +1,33 @@
-# OpenRA Chocolatey Package
-Automatically install and update using the [Chocolatey](https://github.com/chocolatey) binary package panager for Windows.
+# OpenRA Chocolatey Packages
+
+Install and update using the [Chocolatey](https://github.com/chocolatey) package manager for Windows. Because even the stable releases are considered "prerelease" due to the NuGet [version scheme](http://docs.nuget.org/Create/Versioning#prerelease-versions), we have to release two packages.
```

Comment:
I can update that now or you can tweak it if the PR is acceptable. The README isn't part of the Chocolatey packages, at any rate.